src/component/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[];
+}
+
+export const Tabs: React.FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams();
+  const selectedId = tabs.find(tb => tb.id === tabId);
```

Comment:
```suggestion
  const selectedTab = tabs.find(tb => tb.id === tabId);
```
Actually, it's a tab, not id